### ISSUE TITLE
Does not work properly on Switch OLED.

### ISSUE THREAD

**Issue by tma533026** [TTghostsemir/fastCFWswitch#1]
My config is set up and corresponds to Hekate_ipl.ini. But after using only reboot to Hekate. And no payload!

On Switch OLED.

**Comment by TTghostsemir** [comment]
Oh😮

Let me test it with my Oled later.

I'll probably find a way 😉

**Comment by tma533026** [comment]
Thank you for your hard work.

**Comment by AK478BB** [comment]
> Thank you for your hard work.

I tested it on my OLED's  without any problem, may be you need to set /config/fastcfwswitch/config.ini correctly, although it's only reboot to Hekate on Mariko.

**Comment by tma533026** [comment]
> > Thank you for your hard work.
> 
> I tested it on my OLED's without any problem, may be you need to set /config/fastcfwswitch/config.ini correctly, although it's only reboot to Hekate on Mariko.

After 6 hours of setting, I have determined that there is no problem with the setting.
If it's just for Mariko to reboot on Hekate.
Well my question can ignore it, but no payload goes into the set options!
Still thanks for your reply.

**Comment by AK478BB** [comment]
Of course,  take a look at the author mentioned,  it is with Mariko consoles so that the only payload.bin injectet in the root! 
You can go back into Hekate through fastcfwswitch in mariko consoles.

**Comment by TTghostsemir** [comment]
The Problem is that the RTC registers not Resetting on Mariko consoles.

But I am already trying to fix it 😉

**Comment by AK478BB** [comment]
> The Problem is that the RTC registers not Resetting on Mariko consoles.
> 
> But I am already trying to fix it 😉

I don't think the mariko can be implement the function of switching options after reboot as same as the erista's.  That vulnerable exploit was fixed in mariko's hardware already.  
Someone has recompiled sysmodule.ovl to support the function of restarting and power off in mariko and erista.  The more realistic implementation is compile fastcfwswitch to change autoboot options in hekate_ipl.ini although it's impossible.
